tests/Fixtures/e2e/standard_script.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ $PHPDBG=1 ]]
+if [ "$PHPDBG" = "1" ]
```

Comment:
what was the issue? on linux it works fine

Reply:
on macos it was always `true`

Reply:
I couldn't reproduce it, might be a different shell (I'm using zsh), but in any case it's better to compare strings in bash

Reply:
`if [[ $PHPDBG=1 ]]` does not work on Terminal app in MacOS (while works on zsh). See image below. Also, `if [[ $PHPDBG=1 ]]` is not the same as `if [[ $PHPDBG = 1 ]]`


<img width="421" alt="phpdbg" src="https://user-images.githubusercontent.com/3725595/35823481-bf89b18c-0ac0-11e8-8086-82fb4fdb9264.png">